Replace valloc() with memalign().

valloc() no longer exists in some toolchains like Android arm64, but using memalign(getpagesize(), size) should be identical.

https://github.com/gnustep/libobjc2/pull/91
diff --git a/block_to_imp.c b/block_to_imp.c
index f70c1c7..fc92c8b 100644
--- a/block_to_imp.c
+++ b/block_to_imp.c
@@ -87,6 +87,11 @@ static int mprotect(void *buffer, size_t len, int prot)
 
 	return 0 != VirtualProtect(buffer, len, newProt, &oldProt);
 }
+#else
+static void *valloc(size_t len)
+{
+	return memalign(getpagesize(), len);
+}
 #endif // _WIN32
 
 
